packages/pg-pool/test/bring-your-own-promise.js

```diff
@@ -8,29 +8,34 @@ const BluebirdPromise = require('bluebird')
 
 const Pool = require('../')
 
-const checkType = promise => {
+const checkType = (promise, expectError) => {
   expect(promise).to.be.a(BluebirdPromise)
-  return promise.catch(e => undefined)
+  return promise.catch(e => {
```

Comment:
Note for reviewer:  It took me a while to figure out why this test kept failing with an error about the value being undefined, so I added the new argument so we can differentiate an unexpected error scenario and subsequently report it as a test failure.

Reply:
nice - thanks.  As an aside, I think it's probably time I deprecate BYOP for pg@9.0 as all current supported versions of node include a native promise.